### PR TITLE
Tech writer review

### PR DESCRIPTION
**Pull request by DannaShirley**

### CODE REVIEW

**Comment by DannaShirley** [comment]
Tech writing review completed.

**Comment by DannaShirley** [comment]
There are a few open items to confirm/update: 
* Potential global change: References to GitHub GenesysAppFoundryrepo may need to change to the Blueprints repo.
* Line 21: Agnes confirmed that they will be downloading the integration from AppFoundry, but this seems unclear to me.
* Lines 97-99 may not render properly without HTML code. Please verify.

File: blueprint/index.md [hunk 1]
```diff
@@ -1,102 +1,114 @@
...
-* **Amazon Translate** - Amazon Translate is a translation service that delivers fast, high-quality, and affordable language translation. Amazon Translate is the translate service used in the Chat Translator solution.
 
-### Software Development Kit (SDK)
-* **Genesys Cloud Platform API SDK** - This SDK is used for the initial interaction of agent and customer over chat.
-* **Amazon Web Services SDK** - This SDK enables developers to build libraries and applications that use AWS services. You can use the JavaScript API in the browser and inside Node.js applications on the server.
+* **Genesys Cloud** - A suite of Genesys cloud services for enterprise-grade communications, collaboration, and contact center management. You deploy the Chat Translator solution in Genesys Cloud.
+* **Genesys AppFoundry** - The Genesys app marketplace for solutions that run on the Genesys Cloud platform. You download the integration used in this solution from the Genesys AppFoundry.
```
Comment (AgnesCorpuz) [inline blueprint/index.md:21]: The developers can download the template files in this repository from the AppFoundry by clicking "Get It Now" on the blueprint. You may view the [Genesys Cloud Chat Assistant Blueprint](https://appfoundry.genesys.com/filter/genesyscloud/listing/7ea62ddd-e183-42ce-9ba9-ee1f37370660) from the AppFoundry that I previously authored as an example.

image

File: blueprint/index.md [hunk 2]
```diff
@@ -1,102 +1,114 @@
...
+
+   ![Interaction Widget name](images/name-interaction.png "Interaction Widget name")
 
-   <pre class="language-nohighlight"><code class="language-nohighlight">https://localhost/?conversationid=&#123;&#123;pcConversationId&#125&#125;&language=&#123;&#123;pcLangTag&#125&#125;</code></pre>
+4. Click the **Configuration** tab.
+5. In the **Application URL** box, type the URL of the web application. Be sure to specify the full URL, beginning with `https:`.
 
-   **(Optional)** Select a group for filtering user acces to the widget.
+   ```
+   https://localhost/?conversationid={{pcConversationId}}&language={{pcLangTag}}
```
Comment (AgnesCorpuz) [inline blueprint/index.md:98]: We will need to use the original format to show this line of code in the Developer Center. Having a {{ }} in our markdown will cause issues in the site itself and will not be able to render it properly.

File: blueprint/index.md [hunk 3]
```diff
@@ -117,55 +129,45 @@ The Chat Translator integration has the following stages:
...
 
-Clicking it will open the Chat Translator. When the customer sends a message in a foreign language, the Chat Translator automatically detects the language and translates it to the agent's configured language in Genesys Cloud.
-![Translate Chat](images/chat-translate.png "Translate Chat")
-
-Clicking the **Open Canned Responses** above the message box allows an agent to send canned response translated to the customer's language.
-![Translate Canned Response](images/translate-canned-response.png "Translate Canned Response")
-
-## Additional Resources
+   :::primary
+   **Important**: Make sure the Developer Center URL matches the region where your  Genesys Cloud organization is located. For more information, see the [Access the developer tools](https://developer.mypurecloud.com/gettingstarted/developer-tools-intro.html#accessTools "Goes to the Access the developer tools section on the Developer tools quick start page") section on the Developer tools quick start page.
```
Comment (AgnesCorpuz) [inline blueprint/index.md:154]: I think there is double space between "your" and "Genesys Cloud"

**Review by AgnesCorpuz — APPROVED** [review]
I think overall these are good edits! Thank you so much Danna! I have a few comments and  clarifications as you commented in your Pull Request.

I will approve and merge your changes and will just edit the ones that needs updating on my end.

Again, thank you so much!

**Pull request merged**